src/oop/oop2.py

```diff
@@ -19,6 +23,14 @@ def __init__(self, num_wheels):
 
 # TODO
 
+class Motorcycle(GroundVehicle):
+    def __init__(self, num_wheels=2):
+        self.num_wheels = num_wheels
+
```

Comment:
since motorcycle is subclass of ground vehicle, you probably want to instantiate # of wheels to constructor using superclass,
super().__init__(2)